relational_operators/SelectOperator.hpp

```diff
@@ -241,15 +286,18 @@ class SelectWorkOrder : public WorkOrder {
                   std::vector<attribute_id> &&simple_selection,
                   const std::vector<std::unique_ptr<const Scalar>> *selection,
                   InsertDestination *output_destination,
-                  StorageManager *storage_manager)
+                  StorageManager *storage_manager,
+                  const int numa_node = 0)
```

Comment:
@zuyu the purpose of this PR is to make the select operator partition aware. i.e. The select work units will be executed on the NUMA nodes where the partitions are present. So, if we remove this, then this PR would have no effect.